bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/WriteAdvHandle.java

```diff
@@ -45,12 +45,36 @@
      *
      * @param entryId entryId to be added
      * @param data array of bytes to be written
-     * @return an handle to the result, in case of success it will return the same value of param entryId
+     * @return an handle to the result, in case of success it will return the same value of param entryId.
```

Comment:
It's recommended to have no full stop at the end of @param, @return and @throws

http://www.oracle.com/technetwork/articles/java/index-137868.html